web_src/js/modules/toast.js

```diff
@@ -0,0 +1,59 @@
+import {htmlEscape} from 'escape-goat';
+import {svg} from '../svg.js';
+
+const levels = {
+  info: {
+    icon: 'octicon-check',
+    background: 'var(--color-green)',
+    dur: 2000,
+  },
+  error: {
```

Comment:
use case: validation failed for some form.
For complex situations that cannot be checked easily with the browser API, we don't have a way to communicate errors at the moment.

Reply:
While it might not be perfect for that, it is far better than sending the request to the server rejecting it and then reloading the page with an error.